extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -527,14 +527,18 @@ export class ImageViewer {
   onZoom_(scale, deltaX, deltaY, animate) {
     const newScale = this.boundScale_(scale, true);
     if (newScale == this.scale_) {
-      return;
+      return Promise.resolve();
```

Comment:
We should instead allow `!Promise|undefined` return type. If we need to check on the calling side - we can simply do `dev().assert()`